addons/gamification/data/gamification_karma_rank_data.xml

```diff
@@ -15,7 +15,7 @@
             <field name="description" type="html"><p>You just began the adventure! Welcome!</p></field>
             <field name="description_reach_next" type="html"><p>Keep going !</p></field>
             <field name="karma_min">1</field>
-            <field name="image" type="base64" file="gamification/static/img/rank_newbie.png"/>
+            <field name="image" type="base64" file="gamification/static/img/rank_bachelor_badge.svg"/>
```

Comment:
I guess I wanted to write rank_newbie_badge.png ?

Reply:
Indeed !